execution_engine/src/core/runtime_context/mod.rs

```diff
@@ -689,6 +695,7 @@ where
                 // bits are verified from within API call.
                 false
             }
+            Key::SystemContractRegistry => true,
```

Comment:
I'd make this key unreadable by normal read APIs. We have a dedicated set of functions `get_mint()` `get_auction()` etc. on the WASM side. As currently implemented users won't even know what type is that as the `SystemContractRegistry` type alias is in EE and that's fine as WASM doesn't need to know how system contracts are stored.

```suggestion
            Key::SystemContractRegistry => false,
```